{{cookiecutter.__project_name}}/LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) {{cookiecutter.copyright_year}}, Alaska Satellite Facility
+Copyright (c) {{ cookiecutter.copyright_year }}, Alaska Satellite Facility
```

Comment:
Does `cookiecutter.copyright_year` not need the double-underscore prefix like in e.g. `cookiecutter.__project_name`?